its/src/test/java/com/sonar/it/scanner/msbuild/utils/GeneralCommand.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.scanner.msbuild.utils;
+
+import com.sonar.orchestrator.build.BuildResult;
+import com.sonar.orchestrator.util.Command;
+import com.sonar.orchestrator.util.CommandExecutor;
+import com.sonar.orchestrator.util.StreamConsumer;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.sonar.it.scanner.msbuild.utils.TestUtils.LOG;
+import static com.sonar.it.scanner.msbuild.utils.TestUtils.TIMEOUT_LIMIT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GeneralCommand extends BaseCommand<GeneralCommand> {
+  private long timeout = TIMEOUT_LIMIT;
+  private final Command command;
+
+  private GeneralCommand(String command, Path workingDirectory) {
```

Comment:
`workingDirectory` comes second because `create(String command)` and `create(String command, Path workingDirectory)` overloads are more future proof than `create(Path workingDirectory, String command)`. It is annoying when the order of parameters changes in overloads.